get_all_data/get_malware_analyses.py

```diff
@@ -0,0 +1,86 @@
+import csv
+import argparse
+
+from pytx import Malware
+from pytx.vocabulary import Malware as MA
+
+import common_field_transforms as CFT
+from datetime import datetime
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-T", "--sample_type", help="The type of malware sample \
+        for which to search")
+    parser.add_argument("-H", "--share_level", help="The TLP Share Level \
+        to match")
+    parser.add_argument("-t", "--text", help="The text to match against")
+    parser.add_argument("-s", "--status", help="The status (StatusType) \
+        to match")
+    parser.add_argument("-S", "--strict_text", help="Search results will \
+        only match strict text (without wildcards)", action="store_true")
+    parser.add_argument("-e", "--end_date", help="Search for samples \
+        created up until a date (inclusive). The default is today's date. \
+        Note that this date is converted to UTC time, so for example entering \
+        1-27-16 would search until 1-27-16 00:00:00 UTC.",
+        type=str, default=str(datetime.utcnow()))
+    parser.add_argument("-d", "--days_back", help="Number of days prior to the \
+        end_date from which results will be pulled.", type=int, default=1)
+    return parser.parse_args()
+
+def main():
+    s = get_args()
+    format_ = '%d-%m-%Y'
+    for day_counter in range(s.days_back):
+        until_param, until_param_string, since_param, since_param_string = \
+            CFT.get_time_params(s.end_date, day_counter, format_)
+
+        output_file = 'malware_analyses_' + since_param_string + '_to_' + \
+            until_param_string + '.csv'
+        with open(output_file,'wb') as fout:
+            writer = csv.writer(fout)
+
+            # TODO: Remove this once querying the fields related to a sample
```

Comment:
make fetching the full sample a command line arg option as well, we can return an error if it's enabled for now.